include/rcutils/strerror.h

```diff
@@ -23,7 +23,7 @@ extern "C"
 #include "rcutils/visibility_control.h"
 
 /// Retrieve the string corresponding to the last system error.
-/* This function retrieves the value of errno, and calls the system-specific
+/** This function retrieves the value of errno, and calls the system-specific
```

Comment:
Actually this still doesn't follow the style... sorry.

It should be:

```
/// Brief ... description.
/**
 * Start detailed description here.
 */
...
```

Reply:
This should be fixed now.